scala-package/infer/src/main/scala/org/apache/mxnet/infer/javaapi/ObjectDetector.scala

```diff
@@ -113,6 +113,14 @@ object ObjectDetector {
     org.apache.mxnet.infer.ImageClassifier.loadImageFromFile(inputImagePath)
   }
 
+  def reshapeImage(img : BufferedImage, newWidth: Int, newHeight: Int): BufferedImage = {
+    org.apache.mxnet.infer.ImageClassifier.reshapeImage(img, newWidth, newHeight)
```

Comment:
So I'm not sure whether or not we should add these to the public API for ObjectDetector. I made this mistake with loadImageFromFile earlier apparently because we don't yet have ImageClassifier in the Java API.

In the Scala version, the objectDetector benchmark and example call loadImageFromFile, reshapeImage, and bufferedImageToPixels directly from the ImageClassifier class. We're not able to do that from here yet but even if we could it seems odd to do so. Maybe we should move them to a utils class or something? What's everyone else think about how we should handle this?

Reply:
@andrewfayres I am expecting this: https://github.com/apache/incubator-mxnet/blob/master/scala-package/core/src/main/scala/org/apache/mxnet/Image.scala would eventually help and replace the current JavaIO we have.